modules/og_context/config/install/og_context.ogcontextneogation.yml

```diff
@@ -0,0 +1 @@
+og_context:
```

Comment:
There is a typo here: `neogation` instead of `negotation`. It's probably also not necessary to prefix the config ID with `ogcontext` because the config system already automatically provides the prefix `og_context.`.